app/app.js

```diff
@@ -46,17 +48,29 @@ function initialSetup() {
   $('#closeAll').click(() => {
       $('.gct-folder').removeClass('gct-folder-open');
   });
+
```

Comment:
Could you add this function here? So in this way we open automatically when the file is clicked:

```js
  $('.gct-file-name').click(obj => {
    var href = $(obj.currentTarget)[0].getAttribute("href");
    var file = $(`.file-info > a[href="${href}"]`).parent().parent().parent();
    if ($(file).hasClass('open Details--on')) {
      $(file).removeClass('open Details--on');
    }
  });

```

Reply:
That's a great idea, more useful. 👍